src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -176,27 +178,68 @@ export const CustomerInstitutionMetadataForm = ({
-              {({ field }: FieldProps<boolean>) => (
-                <div>
-                  <FormLabel component="legend">{t('common.nvi')}</FormLabel>
-                  <FormControlLabel
-                    label={t('basic_data.institutions.institution_is_nvi_applicable')}
-                    control={
-                      <Checkbox
-                        data-testid={dataTestId.basicData.institutionAdmin.nviInstitutionCheckbox}
-                        {...field}
-                        checked={field.value}
-                      />
-                    }
-                  />
-                </div>
-              )}
-            </Field>
+            <div>
+              <Field name={CustomerInstitutionFieldNames.NviInstitution}>
+                {({ field }: FieldProps<boolean>) => (
+                  <>
+                    <FormLabel component="legend" sx={{ fontWeight: 'bold' }}>
+                      {t('common.nvi')}
+                    </FormLabel>
+                    <FormControlLabel
+                      label={t('basic_data.institutions.institution_is_nvi_applicable')}
+                      control={
+                        <Checkbox
+                          data-testid={dataTestId.basicData.institutionAdmin.nviInstitutionCheckbox}
+                          {...field}
+                          checked={field.value}
+                          onChange={(event, checked) => {
+                            if (!checked) {
+                              setFieldValue(CustomerInstitutionFieldNames.RboInstitution, false);
+                            }
+                            field.onChange(event);
+                          }}
+                        />
+                      }
+                    />
+                  </>
+                )}
+              </Field>
+
+              <Field name={CustomerInstitutionFieldNames.RboInstitution}>
+                {({ field }: FieldProps<boolean>) => (
+                  <>
+                    <FormLabel
+                      component="legend"
+                      sx={{
+                        fontWeight: 'bold',
+                        marginLeft: '2rem',
+                      }}>
+                      {t('common.rbo')}
+                    </FormLabel>
+                    <FormControlLabel
+                      label={t('basic_data.institutions.institution_receives_funding_via_rbo')}
+                      disabled={!values.customer.nviInstitution}
+                      control={
+                        <Checkbox
+                          sx={{
+                            marginLeft: '2rem',
+                          }}
+                          data-testid={dataTestId.basicData.institutionAdmin.rboInstitutionCheckbox}
+                          {...field}
+                          checked={field.value}
+                        />
+                      }
+                    />
+                  </>
```

Comment:
Her kan vi også forenkle litt, ved å oppgi innrykk en plass i stedet for to. Stor fordel når sånne ting skal endres senere at det ligger bare en plass, for å unngå at de kommer ut av sync.


```suggestion
                  <Box sx={{ ml: '2rem' }}>
                    <FormLabel
                      component="legend"
                      sx={{ fontWeight: 'bold' }}>
                      {t('common.rbo')}
                    </FormLabel>
                    <FormControlLabel
                      label={t('basic_data.institutions.institution_receives_funding_via_rbo')}
                      disabled={!values.customer.nviInstitution}
                      control={
                        <Checkbox
                          data-testid={dataTestId.basicData.institutionAdmin.rboInstitutionCheckbox}
                          {...field}
                          checked={field.value}
                        />
                      }
                    />
                  </Box>
```